### DISCUSSION TITLE
WellUe Heart Rate and SPO2 Measurement

### DISCUSSION THREAD

**Discussion by jones139** [OpenSeizureDetector/Android_Pebble_SD#117] [04_Devices]
A separate wrist/hand worn Heart Rate and SPO2 measurement device could improve performance of the measurements using these devices (See #114). @44616E suggested:

The wellue devices we have tried so far

- Kids ring https://getwellue.com/pages/kidso2-child-oxygen-monitorWas good, alittle small got knocked off a few times. Was lost on the way to the emergency department. But did end up finding it a month later, so its now a spare. 
- There is an adult version. Checkme O2 https://getwellue.com/products/checkme-o2-max-wrist-pulse-oximeterVery long battery life (70 hours of recording), and doesn't fall off. Solid hr and O2 readings all night. 

Both also have an accelerometer but the data is just magnitude and unitless and 1Hz.

This project shows how to read the wellue rings seems relatively easy surprisingly. https://github.com/ecostech/viatom-ble

**Comment by 44616E** [comment]
I have been able to collect data from both devices in near realtime with python with some minor mods to https://github.com/z3ugma/o2r/tree/master. Next step is to port to Android and work out how to incorporate it into OSD.
